### ISSUE TITLE
[BUG]Error and doesn't load in 1.17

### ISSUE THREAD

**Issue by trevor1097** [Silthus/sLimits#26]
**Describe the bug**
Doesn't load in 1.17

**To Reproduce**
1. Add plugin to server
2. start server and plugin error
3. doesn't load
<details>
<summary>Console Errors</summary>

```[10:45:18] [Server thread/INFO]: [slimits] Enabling slimits v1.8.0
[10:45:18] [Server thread/ERROR]: [slimits] Error while starting lifecycle.
java.lang.NullPointerException: Cannot invoke "com.netflix.governator.lifecycle.LifecycleManager.start()" because "this.lifecycleManager" is null
	at net.silthus.slimits.slib.injection.SpigotExtension.start(SpigotExtension.java:90) ~[?:?]
	at net.silthus.slimits.slib.injection.AbstractSpigotExtension.onEnable(AbstractSpigotExtension.java:56) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:518) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:432) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:276) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1118) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at java.lang.Thread.run(Thread.java:831) [?:?]
[10:45:18] [Server thread/ERROR]: Error occurred while enabling slimits v1.8.0 (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "com.netflix.governator.guice.LifecycleInjector.createInjector(com.google.inject.Module[])" because "this.lifecycleInjector" is null
	at net.silthus.slimits.slib.injection.SpigotExtension.start(SpigotExtension.java:95) ~[?:?]
	at net.silthus.slimits.slib.injection.AbstractSpigotExtension.onEnable(AbstractSpigotExtension.java:56) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:518) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:432) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:276) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1118) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at java.lang.Thread.run(Thread.java:831) [?:?]
[10:45:18] [Server thread/INFO]: [slimits] Disabling slimits v1.8.0
[10:45:18] [Server thread/ERROR]: Error occurred while disabling slimits v1.8.0 (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "com.netflix.governator.lifecycle.LifecycleManager.close()" because "this.lifecycleManager" is null
	at net.silthus.slimits.slib.injection.SpigotExtension.tearDown(SpigotExtension.java:102) ~[?:?]
	at net.silthus.slimits.slib.injection.AbstractSpigotExtension.onDisable(AbstractSpigotExtension.java:62) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:405) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:533) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:374) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:518) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:432) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:276) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1118) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[patched_1.17.jar:git-Paper-"28aacb4"]
	at java.lang.Thread.run(Thread.java:831) [?:?]
```
</details>
**Expected behavior**
Plugin Loads

**Server Information (please complete the following information):**
- Server version (`/bukkit:version`): git-Paper-"28aacb4" (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT)
- Installed plugins (`/bukkit:plugins`): BottledExp, BuycraftX, ChestShop, ChestShop-towny, Citizens, CMI, CMILib, CombatLogX, CoreProtect, DeathChestPro, DeluxeMenus, DiscordSRV, dynmap*, Dynmap-Towny, Jobs, LuckPerms, mcMMO, MorphRedeem, MOTD, Multiverse-Core, Multiverse-NetherPortals, PlaceholderAPI, ProtocolLib, Quests, SilkSpawners, SirBlobmanCore (SirBlobmanAPI, XSeries), slimits, Towny, TownyCamps, TownyChat, TownyFlight, Vault, VoidGenerator, Votifier, WorldBorder, WorldEdit, WorldGuard, WorldGuard-Towny*, WorldGuardExtraFlags
- WorldGuard Report Pastebin Link (`/wg report -p`):

**Comment by Silthus** [comment]
I will update sLimits to 1.17 once I am back from my vacation (~2 weeks).

**Comment by Silthus** [comment]
Resolved with the 2.0.0 release.
